README.md

```diff
@@ -49,7 +49,9 @@ var isEnabled = normalize(['string', 'boolean'], true);
 // Provide a function as first argument to do custom coercion
 var now = new Date();
 var enabledSince = normalize(function(value) {
-  return value.constructor === Date ? value : null;
+  if (value.constructor === Date) {
```

Comment:
@erikkemperman sorry, I didn't mean it about ternaries; I meant that the result of a function call is `undefined` if it doesn't return anything explicitly.  I just don't like the look/mental parse-ability of ternaries (especially ones that return `undefined` as the "else" case.